packages/codemods/package.json

```diff
@@ -31,6 +31,7 @@
     "tslib": "^2.1.0",
     "react": "17.0.2",
     "ts-morph": "^10.0.1",
+    "typescript": "5.3.3",
```

Comment:
had to lock ts to 5.3.3 for this package, as `ts-morph` has issue with 5.4: https://github.com/dsherret/ts-morph/issues/1513. 

It was fixed in v22, however in this package current version is v10. If necessary, can be solved separately.